src/contour/ContourGuiApp.cpp

```diff
@@ -371,7 +371,7 @@ TerminalWindow* ContourGuiApp::newWindow(contour::config::Config const& _config)
     auto const* profile = _config.profile(profileName());
     if (!profile)
         return nullptr;
-
+    ContourGuiApp::_config = _config;
```

Comment:
hm, okay, that's my bad for sure. We have multiple coding styles, sadly, and it's all my fault. I am sorry for that. It's probably best to simply rename the function parameter's `_config` to `config` and then have this line like `_config = config`.